fog/sql_recovery_db/cleanup/src/config.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Specifies configuration for the Fog SQL Recovery DB cleanup binary.
+
+use clap::Parser;
+use serde::Serialize;
+
+/// Configuration parameters for the Fog SQL recovery DB cleanup task.
+#[derive(Clone, Parser, Serialize)]
+#[clap(version)]
+pub struct SqlRecoveryDbCleanupConfig {
+    /// If set to true, performs cleanup task for unused egress keys.
+    #[clap(long)]
+    pub egress_keys: bool,
```

Comment:
maybe this should be the other way, so, cleanup of these should happen unless you opt out explicitly? I think usually you would want to do this

Reply:
I would normally agree, but James mentioned that this tool might be used for other types of DB cleanups. In that scenario, there might be multiple types of cleanups, and I think it might be better to have the Ops engineer specify exactly what type of cleanup they want. That said, happy to change it this perspective isn't convincing.